vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/util/yaml/decoder_test.go

```diff
@@ -197,8 +197,8 @@ stuff: 1
 		t.Fatal("expected error with yaml: violate, got no error")
 	}
 	fmt.Printf("err: %s\n", err.Error())
-	if !strings.Contains(err.Error(), "yaml: line 2:") {
-		t.Fatalf("expected %q to have 'yaml: line 2:' found a tab character", err.Error())
```

Comment:
have you checked what is happening here?

Reply:
Not sure I understand the question. Are you seeing that error in the logs? If so can you point me there because I can't find it.

Reply:
> Not sure I understand the question. Are you seeing that error in the logs? If so can you point me there because I can't find it.

he wants to know the same thing I do.  Why has the line number changed?  Was it consistently incorrect before?  Can you provide an example?

Reply:
TestDecodeBrokenYAML: This same change was actually made by @deads2k in 4.1 (8c90407) although I didn't know it a t the time. So this is really a back port from 4.1. As to the cause, the error message lists the line number of yaml, which is 3, vs line number of json which is 2. I do not know why it was ever 2, perhaps the underlying ymal.v2 impl changed as that is the only diff -> moving to v2.2.4.